src/libs/antares/study/area/list.cpp

```diff
@@ -862,6 +869,7 @@ static bool AreaListLoadFromFolderSingleArea(Study& study,
     area.spreadSpilledEnergyCost = 0.;
 
     bool ret = true;
+    IniFile ini;
```

Comment:
```c++
IniFile ini;
```
We instantiate an `IniFile`, pass it to **loadReserves(...)** but never use it in this scope.
This instantiation actually belong to  **loadReserves(...)**'s scope. 
That way, we don't have to pass **ini** to  **loadReserves(...)**.